src/components/Card.js

```diff
@@ -48,20 +39,22 @@ export default class Card {
     this._likeBtn.classList.toggle("card__like-button_enabled");
   }
 
-  isLiked() {
-    return this._likeArr.some(({ _id }) => {
-      return _id === this._userId;
-    });
+  isLiked(){
+    return this._likeArr.some(({_id}) => {      
+      return _id === this._userId       
+    })
   }
 
-  _renderLikesCounter() {
-    this._likeCounter = this._card.querySelector(".card__like_counter");
-    this._likeCounter.textContent = this._likeArr.length;
+  _renderLikesCounter(){
+    //i cannot move this line to the class constructor, it gets undefined
+    this._likeCounter = this._card.querySelector(".card__like_counter")    
```

Comment:
You can put `this._likeCounter = this._card.querySelector(".card__like_counter")  ` in line 28 inside `_setEventListeners` to find it only **once** (like you did with the heart, trashicon and so on)